### PR TITLE
Release 1.3.3

### PR DESCRIPTION
**Pull request by peterwvj**
Please check whether the CHANGELOG properly reflects recent changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## [1.3.3] - 14.09.2021
+- Support for `KnowledgePanel`s
```
Comment (M123-dev) [inline CHANGELOG.md:2]: We currently just have the models for it, let's announce the KnowledgePanels when we also have the code to use it.

**Review by M123-dev — COMMENTED** [review]
I have one more thing I would like to add before the release, can we wait for it

**Review by monsieurtanuki — APPROVED** [review]
We could probably merge and include #217 too.

File: CHANGELOG.md (hunk 1 above)
Comment (peterwvj) [reply CHANGELOG.md]: OK, I've removed that part

**Comment by peterwvj** [comment]
> We could probably merge and include #217 too.

@monsieurtanuki  feel free to include that too. Once it is merged I'll update the CHANGELOG.

**Comment by peterwvj** [comment]
@M123-dev are you thinking of a particular PR you'd like to have merged? We could just release what we currently have. Publishing to pubdev is just a matter of issuing a single command and I think "small" releases are fine.

**Comment by M123-dev** [comment]
@peterwvj, all things from me are merged, but there are still some errors, am still investigating where they come from.

**Comment by peterwvj** [comment]
@M123-dev that's fine. I merged the changelog changes for now but I didn't publish anything. Let me know how things go

**Comment by M123-dev** [comment]
@peterwvj

stephanegigandet on Slack:
> FYI, we are moving openfoodfacts.net and there are currently issues that make the dart package tests fail.
> Fixing in progress 


The interesting thing is that the test pass on my machine, but not here

**Comment by monsieurtanuki** [comment]
@peterwvj I've just merged #217.

**Pull request merged**